lib/streamlit/testing/element_tree.py

```diff
@@ -1331,15 +1332,17 @@ def get_widget_states(self) -> WidgetStates:
 
         return ws
 
-    def run(self, timeout: float = 3) -> ElementTree:
+    def run(self, timeout: float | None = None) -> ElementTree:
         """Run the script with updated widget values.
-        Timeout is a number of seconds.
+        Timeout is a number of seconds, or None to use the default.
```

Comment:
nit: These methods are also targeted for usage by external users, or? If that is the case, maybe it would be better to do a proper docstring that includes the parameters and return value in the numpy docstring format. The same would also apply to the other methods below that we would count as part of our public API.